charts/common/Chart.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v2
+name: common
+description: A common Helm chart to use for creating other Helm charts
```

Comment:
- I would add "Bedag" in the description

have a look at https://github.com/bedag/helm-charts/blob/master/charts/manifests/Chart.yaml and add
- annotations
- sources
- icon
- remove "home"